RedHatAI/Llama-3.3-70B-Instruct-FP8-dynamic/accuracy/tasks.yml

```diff
@@ -0,0 +1,66 @@
+tasks:
+  - name: arc_challenge
+    metrics:
+      - name: acc_norm,none
+        value: 0.5196
+
+  - name: gsm8k
+    metrics:
+      - name: exact_match,strict-match
+        value: 0.9492
+
+  - name: hellaswag
+    metrics:
+      - name: acc_norm,none
+        value: 0.8643
+
+  - name: mmlu
+    metrics:
+      - name: acc,none
+        value: 0.8131
+
+  - name: truthfulqa_mc2
+    metrics:
+      - name: acc,none
+        value: 0.6321
+
+  - name: winogrande
+    metrics:
+      - name: acc,none
+        value: 0.8453
+
+  # - name: leaderboard_ifeval
```

Comment:
can you add a comment on why these sections are commented out?

general comment applies to the other files as well.

Reply:
I think this is due to the output for the leaderboard accuracy is not consolidated therefore cannot compare with what's on the model card.